src/software/ai/hl/stp/tactic/pass_defender/pass_defender_fsm.cpp

```diff
@@ -79,5 +79,31 @@ void PassDefenderFSM::interceptBall(const Update& event)
             AutoChipOrKick{AutoChipOrKickMode::OFF, 0},
             TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT, 0.0,
             event.common.robot.robotConstants(), std::optional<double>()));
+
+        return;
+    }
+    // something is wrong when this happens,
+    // in this special case, the robot should move away from the ball
+    // as to not violate the rule about covering more than 80% of the ball
+
+    // to avoid dividing by 0
+    Angle face_ball_orientation;
+    if ((ball.position() - robot_position).length() == 0)
+    {
+        face_ball_orientation = Angle().zero();
+    }
+    else
+    {
+        face_ball_orientation = (ball.position() - robot_position).orientation();
     }
+
+    // backup by the length of the ball
+    Point backup_position = Point(robot_position.x() - ROBOT_MAX_RADIUS_METERS,
+                                  robot_position.y() - ROBOT_MAX_RADIUS_METERS);
```

Comment:
I think it would be better to change the backup position relative to the velocity of the ball:

```suggestion
    Point backup_position = robot_position + ball.velocity().normalize(ROBOT_MAX_RADIUS_METERS)
```

Reply:
implemented